torch/csrc/profiler/collection.cpp

```diff
@@ -402,6 +402,10 @@ struct StealOrDefault {
   explicit StealOrDefault(T& container)
       : container_{container}, it_{container.begin()} {}
 
+  StealOrDefault(const StealOrDefault&) = delete;
```

Comment:
are these specific classes called out because they are marked as explicit?

Reply:
Because the member is reference and they are not used. The easiest way is to delete them.